crates/red_knot_python_semantic/resources/mdtest/conditionals.md

```diff
@@ -0,0 +1,15 @@
+# Conditionals
+
+## Narrowing
+
+### `is != None`
+
+```py
+x = None if flag else 1
+y = 0
+if x != None:
+    y = x
+
+reveal_type(x)  # revealed: None | Literal[1]
+reveal_type(y)  # revealed: Literal[0, 1]
```

Comment:
With the new test framework, we can now write this in a shorter way, without using additional variables:
```suggestion
x = None if flag else 1

if x != None:
    reveal_type(x)  # revealed: Literal[1]

reveal_type(x)  # revealed: None | Literal[1]
```